my-app/src/components/home page/events.js

```diff
@@ -14,9 +14,7 @@ class Events extends Component {
     };
   }
 
-  componentDidMount() {
```

Comment:
Why did you get rid of this line?

Reply:
Seconded, not sure why this was removed.

Reply:
I only removed it. Because npm run client would crash when I had it. I just forgot to add it back